src/coreclr/jit/codegenarm64test.cpp

```diff
@@ -6946,6 +6946,60 @@ void CodeGen::genArm64EmitterUnitTestsSve()
     // IF_SVE_JK_4B
     theEmitter->emitIns_R_R_R_R(INS_sve_st1b, EA_SCALABLE, REG_V6, REG_P3, REG_R0, REG_V4,
                                 INS_OPTS_SCALABLE_D); // ST1B    {<Zt>.D }, <Pg>, [<Xn|SP>, <Zm>.D]
+
+    // IF_SVE_ID_2A
+    theEmitter->emitIns_R_R_I(INS_sve_ldr, EA_SCALABLE, REG_P1, REG_R5, 0); // LDR     <Pt>, [<Xn|SP>{, #<imm>, MUL VL}]
+    theEmitter->emitIns_R_R_I(INS_sve_ldr, EA_SCALABLE, REG_P1, REG_R5, 76);   // LDR     <Pt>, [<Xn|SP>{, #<imm>, MUL
+                                                                               // VL}]
+    theEmitter->emitIns_R_R_I(INS_sve_ldr, EA_SCALABLE, REG_P1, REG_R5, -25);  // LDR     <Pt>, [<Xn|SP>{, #<imm>, MUL
+                                                                               // VL}]
+    theEmitter->emitIns_R_R_I(INS_sve_ldr, EA_SCALABLE, REG_P1, REG_R5, -256); // LDR     <Pt>, [<Xn|SP>{, #<imm>, MUL
+                                                                               // VL}]
+    theEmitter->emitIns_R_R_I(INS_sve_ldr, EA_SCALABLE, REG_P1, REG_R5, 255);  // LDR     <Pt>, [<Xn|SP>{, #<imm>, MUL
+                                                                               // VL}]
+
+    // IF_SVE_JG_2A
+    theEmitter->emitIns_R_R_I(INS_sve_str, EA_SCALABLE, REG_P3, REG_R1, 0); // STR     <Pt>, [<Xn|SP>{, #<imm>, MUL VL}]
+    theEmitter->emitIns_R_R_I(INS_sve_str, EA_SCALABLE, REG_P3, REG_R1, 221);  // STR     <Pt>, [<Xn|SP>{, #<imm>, MUL
+                                                                               // VL}]
+    theEmitter->emitIns_R_R_I(INS_sve_str, EA_SCALABLE, REG_P3, REG_R1, -117); // STR     <Pt>, [<Xn|SP>{, #<imm>, MUL
+                                                                               // VL}]
+    theEmitter->emitIns_R_R_I(INS_sve_str, EA_SCALABLE, REG_P3, REG_R1, -256); // STR     <Pt>, [<Xn|SP>{, #<imm>, MUL
+                                                                               // VL}]
+    theEmitter->emitIns_R_R_I(INS_sve_str, EA_SCALABLE, REG_P3, REG_R1, 255);  // STR     <Pt>, [<Xn|SP>{, #<imm>, MUL
+                                                                               // VL}] 
+
+    // IF_SVE_IE_2A
+    theEmitter->emitIns_R_R_I(INS_sve_ldr, EA_SCALABLE, REG_V3, REG_R4, 0, INS_OPTS_NONE,
+                              INS_SCALABLE_OPTS_UNPREDICATED); // LDR     <Zt>, [<Xn|SP>{, #<imm>, MUL VL}]
```

Comment:
This isn't exactly the right use of `INS_SCALABLE_OPTS_UNPREDICATED`. Currently it's used where a predicate isn't being used to mask the result. Whereas here, the predicate is being used as the result.

But, once register allocation is added, we can get rid of `INS_SCALABLE_OPTS_UNPREDICATED`.

Therefore, I'm happy with this change.